code/modules/organs/limbs.dm

```diff
@@ -92,6 +92,25 @@
 		germ_level -= 2 //at germ_level == 1000, this will cure the infection in 5 minutes
 
 
+/datum/limb/proc/handle_antibioticsT2()
```

Comment:
I'm making a new one because it doesn't behave the same way. It removes much more germs at higher level infections, but less on lower ones. It also doesn't last long, but that's a different matter.

Reply:
I would still rather its in the same file, but just handles both effects.
I get the intention but a T2 proc isn't the way.